docs/settings.rst

```diff
@@ -554,6 +554,18 @@ Enables EDNS subnet processing, for backends that support it.
 
 Enable globally the LUA records feature
 
+.. _setting-enable-notification-warning:
+
+``enable-notification-warning``
```

Comment:
This name is a bit broad; how about 'enable-no-notification-targets-warning'?

Reply:
changed